aut_docs/Installation_Guide_WindowsOS.md

```diff
@@ -3,9 +3,8 @@
 ### A one-time setup:
 
 - Download Python (Windows x86-64 executable installer version 3.6.5) 
```

Comment:
You should hyperlink this Python word itself, instead of repeating the word again at the end